examples/app_hid_mouse/src/main.c

```diff
@@ -162,7 +162,7 @@ void Endpoint0(chanend_t chan_ep0_out, chanend_t chan_ep0_in)
                     {
                         /* Returns  XUD_RES_OKAY if handled,
                          *          XUD_RES_ERR if not handled,
-                         *          XUD_RES_RST for bus reset */
+                         *          XUD_RES_UPDATE for bus reset */
```

Comment:
This isn't quite right, UPDATE means an update to bus state - another function needs to be called to determine if it's reset or suspend etc.